<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PythonCompiler</class>
 <widget class="QDialog" name="PythonCompiler">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>320</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>540</width>
    <height>320</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>540</width>
    <height>320</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>540</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">PythonCompiler</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>270</y>
     <width>240</width>
     <height>40</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Required Python Version: </string>
   </property>
   <property name="buddy">
    <cstring>selection</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>20</y>
     <width>50</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Mirror: </string>
   </property>
   <property name="buddy">
    <cstring>mirror</cstring>
   </property>
  </widget>
  <widget class="QCheckBox" name="specify_link">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>192</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Specify download link: </string>
   </property>
  </widget>
  <widget class="QCheckBox" name="opt">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>180</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Enable optimizations</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="ssl">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>135</y>
     <width>191</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Build with SSL</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="lto">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>135</y>
     <width>300</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Build with Link-time-optimizations (LTO)</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="shared">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>100</y>
     <width>191</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Enable shared libraries</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>50</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Prefix: </string>
   </property>
   <property name="buddy">
    <cstring>prefix</cstring>
   </property>
  </widget>
  <widget class="QCheckBox" name="specify_c">
   <property name="geometry">
    <rect>
     <x>225</x>
     <y>220</y>
     <width>160</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Specify C Compiler: </string>
   </property>
  </widget>
  <widget class="QComboBox" name="mirror">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>20</y>
     <width>120</width>
     <height>25</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="prefix">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>220</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>/usr</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cc">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>385</x>
     <y>220</y>
     <width>135</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="download">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>60</y>
     <width>320</width>
     <height>25</height>
    </rect>
   </property>
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="selection">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <property name="duplicatesEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>275</y>
     <width>64</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="configuration">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>225</x>
     <y>180</y>
     <width>295</width>
     <height>25</height>
    </rect>
   </property>
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="customize">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>225</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Customize configurations: </string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>specify_link</sender>
   <signal>clicked(bool)</signal>
   <receiver>download</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>272</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>specify_c</sender>
   <signal>clicked(bool)</signal>
   <receiver>cc</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>289</x>
     <y>238</y>
    </hint>
    <hint type="destinationlabel">
     <x>417</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>specify_link</sender>
   <signal>clicked(bool)</signal>
   <receiver>mirror</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>448</x>
     <y>39</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>customize</sender>
   <signal>clicked(bool)</signal>
   <receiver>opt</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>34</x>
     <y>195</y>
    </hint>
    <hint type="destinationlabel">
     <x>60</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>customize</sender>
   <signal>clicked(bool)</signal>
   <receiver>shared</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>customize</sender>
   <signal>clicked(bool)</signal>
   <receiver>lto</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>143</x>
     <y>193</y>
    </hint>
    <hint type="destinationlabel">
     <x>160</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>customize</sender>
   <signal>clicked(bool)</signal>
   <receiver>ssl</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>346</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>customize</sender>
   <signal>clicked(bool)</signal>
   <receiver>configuration</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>189</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>192</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>specify_link</sender>
   <signal>clicked(bool)</signal>
   <receiver>cc</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>414</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>customize</sender>
   <signal>clicked(bool)</signal>
   <receiver>specify_c</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>28</x>
     <y>192</y>
    </hint>
    <hint type="destinationlabel">
     <x>238</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>customize</sender>
   <signal>clicked(bool)</signal>
   <receiver>cc</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>197</y>
    </hint>
    <hint type="destinationlabel">
     <x>400</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>customize</sender>
   <signal>clicked(bool)</signal>
   <receiver>prefix</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>204</y>
    </hint>
    <hint type="destinationlabel">
     <x>85</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>customize</sender>
   <signal>clicked(bool)</signal>
   <receiver>label_3</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>114</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel">
     <x>54</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>specify_link</sender>
   <signal>clicked(bool)</signal>
   <receiver>label_2</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>111</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>364</x>
     <y>32</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
